lib/benchee/benchmark/scenario.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Benchee.Benchmark.Scenario do
+  defstruct [:job_name, :function, :input_name, :input, :run_times,
+             :run_time_statistics, :memory_usages, :memory_usage_statistics]
+
+  @type t :: %__MODULE__{
+    job_name: binary | nil,
+    function: fun | nil,
+    input_name: binary | nil,
+    input: any | nil,
+    run_times: [non_neg_integer] | nil,
+    run_time_statistics: map | nil,
+    memory_usages: [non_neg_integer] | nil,
+    memory_usage_statistics: map | nil
```

Comment:
Statistics has its own type, `Benchee.Statistics.t` :) (for run time and and memory)